core/web/resolver/query.go

```diff
@@ -440,6 +440,25 @@ func (r *Resolver) Config(ctx context.Context) (*ConfigPayloadResolver, error) {
 	return NewConfigPayload(printer.EnvPrinter), nil
 }
 
+// ConfigV2 retrieves the Chainlink node's configuration (V2 mode)
+func (r *Resolver) ConfigV2(ctx context.Context) (*ConfigV2PayloadResolver, error) {
```

Comment:
I haven't worked with graphql much. Does this need to duplicate the logic of `(cc *ConfigController) Dump`? Can they share?

Reply:
I would see these ought to be kept as separate fields.

Reply:
Which fields? I mean, is this a separate route from the standard web api route that is already mapped to Dump? Will that route also provide this data when new config is in effect? (no preference, but If not, then it should explicitly error instead of trying dump)